docs/contributing.md

```diff
@@ -1,9 +1,11 @@
 ## Contributions policy
 
-We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you posted on this). However, we are taking issues and we do want your [feedback]({{urlRoot}}/index#give-us-feedback).
+We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you posted on this). However, we are taking issues and we do want your feedback. Please come and talk to us about the software and the documentation via:
```

Comment:
`we are taking issues` -> `we are accepting issues`?
`we do want your feedback`, we already said `we` just before this, so just `and do want your feedback` might be better?